### PR TITLE
Update tests for Sinon v3

### PR DESCRIPTION
**Pull request by daffl**
This pull request updates the version range and test matrix to support Sinon 3.0.0.

For #107

### CODE REVIEW

**Comment by daffl** [comment]
Got them all passing now. According to the tests, everything is working with Sinon v3 so a patch/minor release should do it 😄

**Comment by luislobo** [comment]
+1

**Comment by Floby** [comment]
:+1: 
This looks good, also I need it :D

File: package.json [hunk 1]
```diff
@@ -23,20 +23,21 @@
...
     "chai": ">=1.9.2 <5",
-    "sinon": "^1.4.0 || ^2.1.0"
+    "sinon": "^1.4.0 || ^2.1.0 || ^3.0.0"
   },
   "devDependencies": {
-    "chai": "^4.0.2",
+    "chai": "^4.1.0",
     "eslint": "^3.19.0",
     "istanbul": "~0.4.5",
+    "lolex": "^2.1.2",
```
Comment (daffl) [inline package.json]: I had to add this to get the tests to pass with the latest Node/npm

**Comment by domenic** [comment]
Thanks for doing all this work! Curious about why lolex is necessary, but whatever...

**Pull request merged**